package.json

```diff
@@ -7,7 +7,8 @@
     "dev": "next dev -p $CONNECT_ACCOUNT_PORT",
     "build": "next build",
     "lint": "eslint \"{src,pages}/**/*.{js,ts,jsx,tsx}\"",
-    "start": "next start -p $CONNECT_ACCOUNT_PORT"
+    "start": "next start -p $CONNECT_ACCOUNT_BUILD_PORT",
```

Comment:
I remember we talked about that and in my mind we ended up with the default `start` port, am I misremembering?

Reply:
We did, but I guessed that vercel injecting the port env var during build time would still require to specify said variable in the build and start script (with the same name as the one defined in the vercel dashboard). Am I wrong?